extensions/wikia/ArticleVideo/bower_components/html5-skin/js/components/deepMerge.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Credit: https://github.com/KyleAMathews/deepmerge
+ */
+
+(function (root, factory) {
+  // Wikia quickfix for issues with Modil.js (require(['deepMerge']) was returning an empty object)
+  window.ooyalaDeepMerge = factory();
+}(this, function () {
+
+  function isMergeableObject(val) {
+    var nonNullObject = val && typeof val === 'object';
+
+    return nonNullObject
+      && Object.prototype.toString.call(val) !== '[object RegExp]'
+      && Object.prototype.toString.call(val) !== '[object Date]'
+  }
+
+  function emptyTarget(val) {
+    return Array.isArray(val) ? [] : {}
+  }
+
+  function cloneIfNecessary(value, optionsArgument) {
+    var clone = optionsArgument && optionsArgument.clone === true;
+    return (clone && isMergeableObject(value)) ? deepmerge(emptyTarget(value), value, optionsArgument) : value
+  }
+
+  function defaultArrayMerge(target, source, optionsArgument) {
+    //console.log("****** utils defaultArrayMerge target: ", target);
+    //console.log("****** utils defaultArrayMerge source: ", source);
+
+    var destination = target.slice();
+    source.forEach(function(e, i) {
+      if (typeof destination[i] === 'undefined') {
+        destination[i] = cloneIfNecessary(e, optionsArgument)
+      } else if (isMergeableObject(e)) {
+        destination[i] = deepmerge(target[i], e, optionsArgument)
+      } else if (target.indexOf(e) === -1) {
+        destination.push(cloneIfNecessary(e, optionsArgument))
+      }
+    });
+    //console.log("****** utils defaultArrayMerge destination: ", destination);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ACommentedCode)